File: app/src/main/java/com/github2136/mvp/ui/view/IMainView.java
package com.github2136.mvp.ui.view;

import com.github2136.base.IBaseMVPView;

/**
 * Created by yb on 2018/10/23.
 **/
public interface IMainView extends IBaseMVPView {
    void getSuccessful(String msg);

    void getFailure(String msg);
}

File: app/src/main/java/com/github2136/mvp/ui/view/IMainView.kt
package com.github2136.mvp.ui.view

import com.github2136.base.IBaseMVPView

/**
 * Created by yb on 2018/11/2.
 */
interface IMainView : IBaseMVPView {
    fun getSuccessful(msg: String?)

    fun getFailure(msg: String?)
}